ast.d

```diff
@@ -148,6 +150,30 @@ OperatorType parseOperatorType(string input)
     }
 }
 
+enum OperatorClass
+{
+    Math,
+    Relational,
+    Logical
+}
```

Comment:
Why do the right / left types only matter for the Math operators? Also, is string concatenation a "Math" operator?